templates/pages/assets/socket.html.twig

```diff
@@ -45,33 +45,196 @@
       __('Position')
    ) }}
 
+	{% if params["several"] is defined %}
 
-   {% set wiring_side %}
-      {% do call('Glpi\\Socket::dropdownWiringSide', [
-         'wiring_side',
+	{{ fields.dropdownNumberField(
+         'to',
+         1,
+         __('To'),
          {
-            'value': item.fields['wiring_side'],
+            'min': 1
          }
-      ]) %}
-   {% endset %}
-   {{ fields.htmlField(
-      'wiring_side',
-      wiring_side,
-      __('Wiring side'),
-   ) }}
+      ) }}
 
-   {% set networkports %}
-      {% do call('Glpi\\Socket::showNetworkPortForm', [
-         item.fields['itemtype'],
-         item.fields['items_id'],
-         item.fields['networkports_id'],
-         params
-      ]) %}
-   {% endset %}
-   {{ fields.htmlField(
-      'networkports_id',
-      networkports,
-      _n('Network port', 'Network ports', get_plural_number()),
-   ) }}
+		{{ fields.textField(
+         'prefix',
+         '',
+         __('Prefix'),
+         {}
+      ) }}
+
+		{{ fields.textField(
+         'suffix',
+         '',
+         __('Suffix'),
+         {}
+      ) }}
+
+		<input type="hidden" name="add_several" value="1">
+
+	{% endif %}
+
+	{% set wiring_side %}
+	{% do call('Glpi\\Socket::dropdownWiringSide', [
+	         'wiring_side',
+	         {
+	            'value': item.fields['wiring_side'],
+	         }
+	      ]) %}
+	{% endset %}
+
+	{{ fields.htmlField(
+	      'wiring_side',
+	      wiring_side,
+	      __('Wiring side'),
+	   ) }}
+
+	{% if params["several"] is not defined %}
+		{{ fields.numberField(
+         'position',
+         item.fields['position'],
+         __('Position')
+      ) }}
```

Comment:
```Position``` and ```NetworkPort``` field are duplicated from creater form

![image](https://user-images.githubusercontent.com/7335054/173523751-bcd8029f-9907-4862-a9ef-5e09c5510071.png)

After creation we have a blank page


```shell
[2022-06-14 09:28:49] glpiphplog.WARNING: Test logger  
[2022-06-14 09:28:49] glpiphplog.WARNING: Test logger  
[2022-06-14 09:49:51] glpiphplog.CRITICAL:   *** Uncaught Exception Error: Call to a member function request() on null in /home/stanislas/Teclib/dev/GLPI/cable_management/src/Socket.php at line 301
  Backtrace :
  src/CommonGLPI.php:655                             Glpi\Socket->showForm()
  ajax/common.tabs.php:107                           CommonGLPI::displayStandardTab()
```
![image](https://user-images.githubusercontent.com/7335054/173523941-85af54c5-ee01-490a-9df4-5ceda773181b.png)